### PR TITLE
Ignore files for Delphi Pascal and Delphi IDE

### PR DESCRIPTION
**Pull request by kotp**
The IDE is included here because it is so integrated into the language.

Operating system and IDE/Editor things generally don't belong in the project
.gitignore file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,42 @@
-*.swp
-.DS_Store
+# Exercism project
 bin/configlet
 bin/configlet.exe
-*.dproj
```
Comment (rpottsoh) [inline .gitignore]: @kotp I think *.dproj should be retained in the .gitignore.  Delphi will generate a new one when the .DPR file is opened by Delphi.

Comment (kotp) [inline .gitignore]: It should be in there, sorted, is it totally gone?

**Comment by rpottsoh** [comment]
LGTM, thanks.

**Comment by kotp** [comment]
Alright... I initiated, so I shouldn't really merge.

**Pull request merged**